tests/datasets/test_egfxset.py

```diff
@@ -0,0 +1,126 @@
+from mirdata.datasets import egfxset
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    data_home = "tests/resources/mir_datasets/egfxset"
```

Comment:
Please add `os.path.normpath` to any hardcoded path (i.e. paths with "/") so your tests run on Windows (see PR #567)